src/azure-cli/azure/cli/command_modules/network/_help.py

```diff
@@ -3963,6 +4170,16 @@
 helps['network service-endpoint policy-definition update'] = """
 type: command
 short-summary: Update a service endpoint policy definition.
+examples:
+  - name: Update a service endpoint policy definition. (autogenerated)
+    text: |
+        az network service-endpoint policy-definition update \\
+        --add communities='12076:5010' \\
+        --name MyServiceEndpointPolicyDefinition \\
+        --policy-name MyPolicy \\
+        --resource-group MyResourceGroup \\
+        --subscription MySubscription
```

Comment:
usually, we don't need this one. @haroldrandom Do you think we need this argument or not?

Reply:
Which one?

Reply:
yes. It's not related with your CI work. Just the content of examples.

Reply:
Should I remove this argument `--subscription` when counting duplicated example via sorted arguments sequence?
Any shortcoming?